db/config.cc

```diff
@@ -997,13 +997,14 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , relabel_config_file(this, "relabel_config_file", value_status::Used, "", "Optionally, read relabel config from file")
     , object_storage_config_file(this, "object_storage_config_file", value_status::Used, "", "Optionally, read object-storage endpoints config from file")
     , minimum_keyspace_rf(this, "minimum_keyspace_rf", liveness::LiveUpdate, value_status::Used, 0, "The minimum allowed replication factor when creating or altering a keyspace.")
-    , auth_superuser_name(this, "auth_superuser_name", value_status::Used, "", 
+    , auth_superuser_name(this, "auth_superuser_name", value_status::Used, "",
```

Comment:
Why this change?

Reply:
Yes, we generally try to limit patches to their scope. One reason is that it reduces effort when backporting (the unrelated change might need extra treatment), another is to preserve `git log` and `git blame` cleanliness, but the real reason is that that's how we did it when working on the kernel.